modules/measurement/budget-calculator/load.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Module Name: Budget Calculator
+ * Description: Allows users to define their own metrics and generate a budget.json file from them.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
```

Comment:
Replace `1,9,9` with `n.e.x.t` instead here and the the rest of the file to make sure when this is released is replaced with the right version.